src/model/utils/modifyselection.js

```diff
@@ -13,6 +13,8 @@ import Range from '../range';
 import { isInsideSurrogatePair, isInsideCombinedSymbol } from '@ckeditor/ckeditor5-utils/src/unicode';
 import DocumentSelection from '../documentselection';
 
+const wordBoundaryCharacters = ' ,.-():\'"';
```

Comment:
Is `'` a word boundary? What about "it's"?

Reply:
It looks that it is (as far as I can test it in textarea and Gnome Text editor:

`it's a trap` - the text will stop: on spaces and on `'`.

Also check this docs: https://www.unicode.org/reports/tr29/#Default_Word_Boundaries.

We might think of expanding this list to full unicode support but I've used the most common ones for the text.

Reply:
Yes it is - check the behovior on textarea or in text editor:

```
it's a trap
```
In the above the cursor will stop on spaces and on `'`.

Also check this docs: https://www.unicode.org/reports/tr29/#Default_Word_Boundaries.

We might think of expanding this list to full unicode support but I've used the most common ones for the text.

Reply:
Hmm... we should expand the list of `boundaryCharacters` with at least: `?|;` and probably some more common interpuntion characters.

Reply:
@wwalc told me that we need to differentiate between "it's" (which is made of two words so the caret should stop after "s") and "Peter's" where it's a single word which should be removed at once.

Reply:
I'm joking of course. Since a single quote is popular in English and e.g. in macOS it's not treated as a word boundary, I'd not treat it as such.